karma.conf.js

```diff
@@ -13,7 +13,8 @@ module.exports = function ( config ) {
       'node_modules/angularjs-slider/dist/rzslider.js',
       'node_modules/angular-mocks/angular-mocks.js',
       'app/**/*.js',
-      'tests/**/*.spec.js'
+      'tests/**/*.spec.js',
+      'tmp/templates.js'
```

Comment:
remove this and use `karma-ng-html2js-preprocessor` instead

here are parts of the config that you want to change as well:
```
files [
...
'app/**/*.tpl.html'
],
preprocessors: {
'app/**/*.tpl.html': ['ng-html2js']
},
ngHtml2JsPreprocessor: {
stripPrefix: 'app/',
moduleName: 'templates-components'
}
```